### PR TITLE
Remove API may change from akka typed

### PR DESCRIPTION
**Pull request by chbatey**
Refs #26897

### CODE REVIEW

**Comment by chbatey** [comment]
I left them on the testkit, should we remove those too?

File: build.sbt [hunk 1]
```diff
@@ -377,7 +376,7 @@ lazy val actorTyped = akkaModule("akka-actor-typed")
       import akka.util.Timeout
       implicit val timeout = Timeout(5.seconds)
     """)
-  .disablePlugins(MimaPlugin)
+  .disablePlugins(MimaPlugin) // remove before RC?
```
Comment (helena) [inline build.sbt]: Perhaps create an issue for this TODO and mark the number here?

**Review by helena — APPROVED** [review]
LGTM, pretty nice to be doing this.

**Review by patriknw — APPROVED** [review]
LGTM, I'm pretty sure we will do a few more changes in the milestones before 2.6.0 but it's ok to remove now because we have not strict promises for milestones.

**Pull request merged**